neurokit2/ecg/ecg_findpeaks.py

```diff
@@ -274,6 +274,9 @@ def _ecg_findpeaks_neurokit(
     qrs = smoothgrad > gradthreshold
     beg_qrs = np.where(np.logical_and(np.logical_not(qrs[0:-1]), qrs[1:]))[0]
     end_qrs = np.where(np.logical_and(qrs[0:-1], np.logical_not(qrs[1:])))[0]
+
+    if len(beg_qrs) == 0:
+        return []
```

Comment:
```suggestion
        return np.array([])
```
I would be inclined to return an empty numpy array so that the return type remains the same

Reply:
Fixed in the new commit